src/AltinnCore/Common/Services/Implementation/DataStudioSI.cs

```diff
@@ -103,7 +107,7 @@ public void UpdateData<T>(T dataToSerialize, Guid instanceId, Type type, string
         {
             string developer = AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext);
             string dataPath = $"{_settings.GetTestdataForPartyPath(applicationOwnerId, applicationId, developer)}{instanceOwnerId}/{instanceId}/data";
-            string formDataFilePath = $"{dataPath}/{dataId}.xml";
+            string formDataFilePath = $"{dataPath}/{dataId}";
```

Comment:
why is fileendings removed?

Reply:
as we are storing in blob storage, @dskogan  mentioned that it's a best practice to save them as binary files and have the filename, type etc as metadata in instance document